CONTRIBUTORS.md

```diff
@@ -97,13 +111,17 @@ The following individuals have contributed code to Galaxy:
 * David Trudgian <dave@trudgian.net> <david.trudgian@utsouthwestern.edu>
 * Nitesh Turaga <nitesh.turaga@gmail.com>
 * Clayton Turner <clayclay911@gmail.com>
+* Jesse c j van Dam <jesse.vandam@wur.nl>
```

Comment:
I think Dannon had the surname right: https://www.wageningenur.nl/en/Persons/JCJ-Jesse-van-Dam-MSc.htm